sycl/doc/conf.py

```diff
@@ -46,13 +46,33 @@
 # The suffix of source filenames.
 source_suffix = ['.rst', '.md']
 
-# Extensions are mostly in asciidoc which has poor support in Sphinx
-exclude_patterns = ['extensions/*']
+exclude_patterns = [
+    # Extensions are mostly in asciidoc which has poor support in Sphinx.
+    'extensions/*',
+
+    # Sphinx complains about syntax errors in these files.
+    'internal-design/DeviceLibExtensions.rst',
+    'internal-design/SYCLPipesLoweringToSPIRV.rst',
+    'internal-design/fpga_io_pipes_design.rst',
+    'internal-design/Reduction_status.md'
```

Comment:
I think we should fix syntax errors and include these documents. @gmlueck, could you file a ticker for that issue, please?

Reply:
Yes, I agree.  I'll do this once we agree on the directory name.